File: python/1_31_afternoon/list.py
aa = [10, 20, 30, 40]
print("aa[-1]은 %d, aa[-2]는 %d" % (aa[-1], aa[-2]))
print(aa[0:3])
print(aa[2:4])
print(aa)
aa[1] = [100, 200]
print(aa)
del (aa[1])
print(aa)
aa[1:2] = [100, 200]
print(aa)
list1 = [1, "안녕하시렵니까?", 10]
print(list1)
list1.append("안녕하오")
print(list1)

File: r/2_12제출/3_4/matrix/matrix/list.R
list1<-list(name='James Seo',address='Seoul',tel='010-8706-4712',pay=500)
list1
list1$name
list1[1:2]
list1$birth <- '1975-10-23'
list1
list1$name <-c('Seojinsu','James Seo')
list1$name
list1$birth<-NULL
list1

#Q1
s<-list(ko=90,ma=100,en=98)
#Q2
s$sc<-100
#Q3
s$en
#Q4
s[2:3]
#Q5
s$en<-95
#Q6
s
(s$ko+s$ma+s$en+s$sc)/length(s)
#Q7
s$sc<-NULL
s
